pkgs/by-name/ep/epy/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, python3Packages, fetchPypi }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "epy";
+  version = "2023.6.11";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "epy_reader";
```

Comment:
```suggestion
    pname = "epy-reader";
```

Reply:
It's a bit confusing, the PyPI package is called epy-reader, but the source is downloadable under the name [epy_reader](https://pypi.org/project/epy-reader/#files).

Reply:
Thanks for checking properly